tests/test_cases/issue_134/test_integers.py

```diff
@@ -16,12 +16,29 @@ def test_integer(dut):
     """
```

Comment:
Could you move both of the tests in this file into test_cocotb/test_handle, then delete the entire issue_134 directory?

Reply:
OK, also moved the "double" tests.